examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,33 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for gRPC client and
+server applications (configured once per binary), and shows what type of
+telemetry data it can produce for certain RPCs with additional CSM Labels. The
+gRPC Client accepts configuration from an xDS Control plane as the default
+address that it connects to is "xds:///helloworld:50051", but this can be
+overridden with the command line flag --server_addr. This can be plugged into
+the steps outlined in the CSM Observability User Guide.
```

Comment:
Is there a link to the CSM Observability User Guide that you can link here?

Reply:
not yet, but we'll have it in a month